test/clout/core_test.cljc

```diff
@@ -1,9 +1,33 @@
 (ns clout.core-test
-  (:import [clojure.lang ExceptionInfo]
-           [java.util.regex PatternSyntaxException])
-  (:require [clojure.test :refer :all]
-            [ring.mock.request :refer [request]]
-            [clout.core :refer :all]))
+  (:import #?@(:clj [[clojure.lang ExceptionInfo]
+                     [java.util.regex PatternSyntaxException]]
+               :cljs [[goog Uri]]))
+  (:require #?@(:clj [[clojure.test :refer :all]
+                      [ring.mock.request :refer [request]]]
+               :cljs [[cljs.test :refer-macros [is are deftest testing use-fixtures]]])
+            [clout.core :refer [route-matches route-compile]]))
+
+#?(:cljs
+   (defn request
+     "Naive implementation of the Ring Mock Request in ClojureScript."
+     ([method uri]
+      (let [uri    (Uri. uri)
+            host   (if-not (empty? (.getDomain uri)) (.getDomain uri) "localhost")
+            port   (.getPort uri)
+            scheme (.getScheme uri)
+            path   (js/encodeURI (.getPath uri))
+            query  (.getQuery uri)
+            request {:server-port    (or port 80)
```

Comment:
This let clause can be done away with, since you're just returning the `request` directly.

Reply:
I agree.